pkg/validations/proxy_validation_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache v2.0 license.
+package validations
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/microsoft/moc/pkg/certs"
+)
+
+func Test_ValidateProxyURL(t *testing.T) {
+	// Empty proxy url
+	_, err := ValidateProxyURL("")
+	expectedResult := "parse \"\": empty url: Invalid Input"
+	if err.Error() != expectedResult {
+		t.Fatalf("Test_ValidateProxyURL test case failed. Expected error %s but got %s", expectedResult, err.Error())
+	}
+
+	// Invalid proxy url
+	_, err = ValidateProxyURL("//akse2e:akse2e@skyproxy.ceccloud1.selfhost.corp.microsoft.com:3128")
```

Comment:
I thought we started fact that we dont want to test external servers for unit tests. You can setup local server for the tests

Reply:
This is invalid url test. I dont need any proxy server to test this.